docker/nightly/centos7/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM centos:latest
+
+LABEL maintainer="Jumping Yang <yzpcreate@qq.com>"
+LABEL readme.md="https://github.com/Microsoft/omi/blob/master/docker/README.md"
+LABEL description="This Dockerfile will install the latest code of OMI."
+
+ARG fork=Microsoft
+ARG branch=master
```

Comment:
I will add a release version for release, and nightly version will be for testing purposes, like https://github.com/PowerShell/PowerShell/tree/master/docker, thanks.